File: 20220913/ompp-0.8.5/opari/tool/Test/test9.c
int main() {
  int i;

#pragma \
 /*--*/ pomp \
 /*--*/ inst \
 /*--*/ init

#pragma pomp inst off

#pragma pomp inst begin(foo)

#pragma omp parallel
      i = 9;

#pragma pomp inst end(foo)

#pragma pomp inst on

#pragma pomp inst finalize

}

File: 20220913/ompp-0.8.5/opari/tool/Test/test9.f90
      program test9

C$POMP  INST
C$POMP& INIT

      !$POMP INST OFF

      !$POMP INST &
      !$POMP BEGIN(foo)

      !$OMP PARALLEL
      i = 9
      !$OMP end PARALLEL

      !$POMP INST END(foo)

      !$POMP INST ON

   !$POMP INSTFINALIZE

      end
